UI/CardScrollUI.gd

```diff
@@ -0,0 +1,71 @@
+extends Control
+
+var card_scene: PackedScene = preload("res://Cards/Card.tscn")
+
+var discard_pile: Array[CardBase] = CardManager.card_container.discard_pile
+var draw_pile: Array[CardBase]    = CardManager.card_container.draw_pile
+var deck: Array[CardBase]         = CardManager.card_container.default_deck
+var cards_to_display: Array[CardBase] = []
+var card_worlds: Array[CardWorld] = []
+var cardworld: CardWorld = null
+
+func _on_button_pressed() -> void:
+	queue_free()
+
+#Pressing escape will return you to the game
+func _input(event) -> void:
```

Comment:
Can we specify the type here, to help with readability?

Reply:
yes you totally can, good catch ! also maybe naming it `event` when we will have an `EventBase` class is maybe not a good idea i think